python/changelog.d/53.feature.md

```diff
@@ -0,0 +1 @@
+Update the OpenAPI specification to the new version on April 22, 2025.
```

Comment:
IMO I would not include a changelog for this change, since there is no actual change to user behavior. We should add a changeset when new functionality is added, a bug is fixed, or behavior is otherwise changed.

To a consumer, this would signal that there is new functionality, since we pulled in an update to the OpenAPI spec, but this is not the case since the wrappers still need to be implemented.

Reply:
Good point! Done.